src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -396,6 +397,9 @@
   <data name="TabSwitcher_NoMatchesText" xml:space="preserve">
     <value>No matching tab name</value>
   </data>
+  <data name="CmdPalActionPrompt" xml:space="preserve">
+    <value>Type a command name...</value>

```

Comment:
Little sad that we need to force a re-localization for this key change

Reply:
Since we are already _not_ using a `uid` for the search box, can we just keep the key names and juggle them different at runtime?

Reply:
I dont see why not. I think this was originally from the first way I tried implementing this, which was a bust.

Does the localization happen multiple times per release? I figured it happened once per release, right before we released

Reply:
We've got a scheduled job that submits strings at 6pm every day :smile: